runtimerevolution/web3-roulette
# [#WEB3-11] React App Deploy Setup

# Description

Deploy the react app to fly io.
- Update vite server config
- Dockerize the app
- Deploy to fly io
- Add app domain in the google auth service

Also include in this task:
- Show error message if the api is unavailable

Ticket: https://runtime-revolution.atlassian.net/browse/WEB3-11

## Type of change

- New feature (non-breaking change which adds functionality)
- This change requires a documentation update

## How Has This Been Tested?

- Build a docker image with the app and run it in a container, make sure the app is up and running
- Access fly.io app dashboard, check monitoring section, and access app domain
- Make sure we can login/logout into the app and see the giveaways page
- Verify that the error message appears when the api is down

Details in the updated readme file.

## Checklist:
